frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/entity/MfaResetResponse.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.di.authentication.frontendapi.entity;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public record MfaResetResponse(@Expose @SerializedName("authorize_url") String authorizeUrl) {}
+;
```

Comment:
Don't need this `;`

Reply:
Have removed this now, my fault for assuming java needs semi colons everywhere 😄